tensorflow/contrib/tensorrt/kernels/trt_engine_op.cc

```diff
@@ -491,8 +491,8 @@ TRTEngineOp::EngineCtxPair& TRTEngineOp::GetEngine(int batch_size,
     }
     TrtUniquePtrType<nvinfer1::ICudaEngine> engine;
     bool convert_successfully = false;
-    VLOG(0) << name() << " Constructing a new engine with batch size "
-            << batch_size;
+    VLOG(0) << "Building a new TensorRT engine for " << name()
```

Comment:
Please change all VLOG(0) to LOG(INFO), since the default VLOG level is 0 and I think no one will set it to be less than 0.

Reply:
Should I change all instances of `VLOG(0)` in the TFTRT code to `LOG(INFO)`?

Reply:
It's fine to change only the ones that is touched by this PR. :)

Reply:
Done